File: test/py/case09_top_level.py
# このファイルは `test/py/case09_top_level.py` のテスト/実装コードです。
# 役割が分かりやすいように、読み手向けの説明コメントを付与しています。
# 変更時は、既存仕様との整合性とテスト結果を必ず確認してください。

def mul3(n: int) -> int:
    return n * 3


value: int = 7

if __name__ == "__main__":
    print(mul3(value))

File: test/cpp/case09_top_level.cpp
// このファイルは `test/py/case09_top_level.py` のテスト/実装コードです。
// 役割が分かりやすいように、読み手向けの説明コメントを付与しています。
// 変更時は、既存仕様との整合性とテスト結果を必ず確認してください。


#include "cpp_module/py_runtime.h"

// このファイルは `test/py/case09_top_level.py` のテスト/実装コードです。
// 役割が分かりやすいように、読み手向けの説明コメントを付与しています。
// 変更時は、既存仕様との整合性とテスト結果を必ず確認してください。

int64 mul3(int64 n) {
    return n * 3;
}

int64 value = 7;

int main() {
    py_print(mul3(value));
    return 0;
}
